pkg/arvo/sys/vane/ames.hoon

```diff
@@ -4397,10 +4682,30 @@
   ::
   ++  ship-state-12-to-13
     |=  old=ship-state-12
-    ^-  ship-state
+    ^-  ship-state-13
     ?:  ?=(%alien -.old)
       old(heeds [heeds.old ~])
     old(corked [corked.old ~])
+  ::
+  ++  state-13-to-14
+    |=  old=ames-state-13
+    ^-  ^ames-state
+    =-  old(peers -)
+    %-  ~(run by peers.old)
+    |=  old=ship-state-13
+    ?:  ?=(%alien -.old)  old
+    old(keens (~(run by keens.old) keen-state-13-to-14))
+  ::
+  ++  keen-state-13-to-14
+    |=  old=keen-state-13
+    ^-  keen-state
+    =-  old(wan wan)
+    %^    (dip-left:(deq:keen-state-13 want) ,wan=((mop @ud want) lte))
```

Comment:
Can we not just `+tap` this to a list of key-value pairs?  I don't think we need the full expressiveness of `+dip`.